collect_app/src/main/java/org/odk/collect/android/notifications/NotificationManagerNotifier.java

```diff
@@ -73,4 +128,13 @@ private boolean allFormsDownloadedSuccessfully(Context context, HashMap<ServerFo
         }
         return true;
     }
+
+    // The application context will give us the system's locale
+    private Resources getLocalizedResources(Context context) {
```

Comment:
I think there are still a few strings in both the auto send and auto download flows that may end up not translated. This is an existing issue however, so I didn't want to dedicate too much time to restructuring everything to fix it. I reckon we just create an issue for this?

Reply:
Yes, please.

Reply:
Issue at #3988